composer.json

```diff
@@ -89,10 +89,10 @@
         "paragonie/constant_time_encoding": "^2.2",
         "phpspec/php-diff": "^1.0",
         "phpunit/phpunit": "^9.5.19",
-        "psr/cache": "^1.0 || ^2.0 || ^3.0",
-        "psr/container": "^1.0",
-        "psr/http-message": "^1.0",
-        "psr/log": "^1.0 || ^2.0 || ^3.0",
+        "psr/cache": "^3.0",
+        "psr/container": "^2.0",
+        "psr/http-message": "^1.0 || ^2.0",
```

Comment:
We could drop version 1 once https://github.com/terminal42/escargot/pull/33 has been merged, however, there are no packages preventing the installation of version 1, so we might as well keep it. @ausi WDYT?